src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final FigureName figureName = FigureName.CIRCLE;
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("\nFigure:\t" + figureName
```

Comment:
Avoid starting the implementation of a method with an empty line. This applies to the `draw` method here.